jekyll/_cci2/runner-manually-upgrading-launch-agent.adoc

```diff
@@ -0,0 +1,141 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Upgrading the CircleCI Machine Runner on Cloud
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Find resources to manually upgrade an already installed CircleCI Machine Runner on linux, windows, or macOS.
+:icons: font
+:toc: macro:
+:toc-tittle:
+
+This page describes how to update the CircleCI **machine runner** on CircleCI cloud
+
+toc::[]
+
+[#check-current-version]
+== Checking current machine runner version
+
+The machine runner version can be checked by using the `--version` flag with the  binary. The default installation directory for the CircleCI machine runner binary is `/opt/circleci/circleci-launch-agent` on Linux as well as macOS, and `C:\Program Files\CircleCI\circleci-launch-agent.exe` on Windows.
+
+[#enable-auto-update]
+== Enabling auto update
+
+The most straightforward way to upgrade an existing CircleCI machine runner is to enable auto update in the configuration file and restart the runner instance. Auto update can be enabled by removing the `disable_auto_update` option from the machine runner configuration file, or by setting `runner.disable_auto_update` to false, then restarting machine runner instance either manually or by the process management used to manage the CircleCI machine runner agent. More details on configuring machine runner can be found in the xref:runner-config-reference[runner configuration reference] documentation.
+
+In some cases an environment variable may have been used to disable the auto update functionality. If the machine runner installation is not updated after enabling the auto update functionality in the configuration file, the environment variable `LAUNCH_AGENT_RUNNER_DISABLE_AUTO_UPDATE` should be set to have a value of `false`, or unset entirely, and the machine runner instance restarted again. 
+
+=== Linux 
```

Comment:
They refer back to "the process management used to manage the CircleCI machine runner agent". Should I remove the "Restart the machine runner instance either manually" and instead have the same note that's in the "[#manually-update-by-downloading-a-new-version-of-the-machine-runner-binary]" section?

Reply:
Ah, ok. In that case, yes perhaps we could add the note. I don't think it has to be in banner format. So, my suggestion would be something like this:

```
. Restart the machine runner instance with the process management used to manage the CircleCI machine runner agent. If process management is not being used to manage the machine runner, the instance will need to be manually stopped using `kill -9` on macOS and Linux systems, or the Task Manager on Windows systems, and then manually restarted. 
```

And if I am following the flow correctly, might I also suggest moving the "In some cases, an environment variable may have been used to disable the auto update functionality..." paragraph at the end of this section, after the Windows command? It might also work well to make it a banner by adding the `NOTE:` syntax. 

Only if this makes sense in terms of flow. I like the idea of having all of the pertinent information as close together as possible, and then a troubleshooting/related tip afterwards - at least that's how I understand the env var info..!